xamarin/Xamarin.Forms
# [iOS] Fix Gesture issue with Shapes

### Description of Change ###

Fix Gestures not working with Shapes on iOS.

### Issues Resolved ### 

- fixes #12577 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12577. Tap on several of the Paths, if a message appears, the test has passed.

### PR Checklist ###
